index.css

```diff
@@ -61,4 +61,4 @@ body {
     border-color: gold;
```

Comment:
For good coding practices, kindly use the hex color format to style the border color. The reason is to keep your CSS consistent.